.github/workflows/build_cmake.yml

```diff
@@ -74,6 +74,7 @@ jobs:
       run: >
         cmake
         --build build
+        --clean-first
```

Comment:
This was added during debugging where I found one of the workarounds; to try out a clean build. Though this is not the solution for the existing errors, I did not want to remove this as the 'clean-first' would make sure all the residual files are removed, and the project is built from scratch. This is just to rule out suspicion that the issues might be due to existing build files. Let me know if I still need to remove this change.

Btw, I am not sure about clean machines because this is what I found:
Using clean build:
![image](https://github.com/user-attachments/assets/44602e8e-977e-4b83-867e-2fa6901b06a3)
Not using clean build:
![image](https://github.com/user-attachments/assets/82e116f1-3b29-4e35-ad82-44091a3db9eb)